SI/Sem.02/Tasks2/Task2.cpp

```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+
+	char symbol;
+	cout << "Enter a symbol: ";
+	cin >> symbol;
+
+
+	if ((symbol >= 65 && symbol <= 90) || (symbol >= 97 && symbol <= 122)) {
+		switch (symbol) {
+
+		case 'a': cout << "That is a part of the latin alphabet and also a vowel."; break;
```

Comment:
You can reuse the cout logic like this
case 1:
case 2:
case 3: cout<<"....."
In each of the cases 1, 2, 3 we will execute the cout